src/main/obsRecorder.ts

```diff
@@ -133,22 +133,51 @@ const setupScene = (monitorIndex: number) => {
 
   // Correct the monitorIndex. In config we start a 1 so it's easy for users. 
   const monitorIndexFromZero = monitorIndex - 1; 
-  console.info("monitorIndexFromZero:", monitorIndexFromZero);
+  console.info("[OBS] monitorIndexFromZero:", monitorIndexFromZero);
   const { physicalWidth, physicalHeight } = displayInfo(monitorIndexFromZero);
 
+  const obsSettings = osn.NodeObs.OBS_settings_getSettings('Video').data;
+
+  // Checks if val is +-1 compare
+  const isClose = (val: number, compare: number) => {
+    return Math.abs(compare - val) <= 1;
+  };
+
+  // Checks if resolution is close enough to monitor, and if so, sets the settingString.
+  const checkRes = (resolution: string) => {
+    const [resW, resH] = resolution.split('x');
+
+    return isClose(parseInt(resW, 10), physicalWidth) && isClose(parseInt(resH, 10), physicalHeight);
+  };
+
+  // TODO: Output should eventually be moved into a setting field to be scaled down. For now it matches the monitor resolution.
+  obsSettings.forEach((subCategory: any) => {
+    subCategory.parameters.forEach((param: any) => {
+      if (!(param.name === 'Base' || param.name === 'Output')) {
+        return;
+      }
+
+      param.values.forEach((vpair: { [key: string]: string }) => {
+        const res = Object.keys(vpair)[0];
+        if (checkRes(res)) {
```

Comment:
What if this is not close? I think we should throw an error and make it very obvious in logs